system-addon/test/functional/mochitest/head.js

```diff
@@ -13,6 +19,18 @@ const ACTIVITY_STREAM_PREF = "browser.newtabpage.activity-stream.enabled";
 pushPrefs([ACTIVITY_STREAM_PREF, true]);
 gBrowser.removePreloadedBrowser();
 
+function setUpActivityStreamTest() { // eslint-disable-line no-unused-vars
+  return (async function() {
+    await PlacesTestUtils.clearHistory();
+    await PlacesUtils.bookmarks.eraseEverything();
```

Comment:
Shouldn't this be done in a cleanup function as well?

Reply:
Added.